pkg/kv/kvclient/kvcoord/rangefeed_message.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2022 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package kvcoord
+
+import "github.com/cockroachdb/cockroach/pkg/roachpb"
+
+// RangeFeedMessage is a type that encapsulates the roachpb.RangeFeedEvent.
+type RangeFeedMessage struct {
+
+	// RangeFeed event this message holds.
+	*roachpb.RangeFeedEvent
+
+	// The span of the rangefeed registration that overlaps with SST span if
+	// event has an underlying roachpb.RangeFeedSSTable event.
```

Comment:
I wouldn't mention anything about SST;
Isn't this span a "range" against which rangefeed was established?